themes/bootstrap3/templates/myresearch/edit.phtml

```diff
@@ -6,11 +6,11 @@
   $this->layout()->breadcrumbs = '<li><a href="' . $this->url('myresearch-home') . '">' . $this->transEsc('Your Account') . '</a></li> <li class="active">' . $this->transEsc('Edit') . '</li>';
```

Comment:
Could the breadcrumb link be fixed as well now that the list id is known?

Reply:
Good idea -- done!